nextcloud_update.sh

```diff
@@ -309,25 +309,22 @@ fi
 # Update all Nextcloud apps
 if [ "${CURRENTVERSION%%.*}" -ge "15" ]
 then
-    # Make sure maintenance:mode isn't activated (it will fail if it is)
-    occ_command maintenance:mode --off
+    occ_command_no_check maintenance:mode --off
     # Check for upgrades
     print_text_in_color "$ICyan" "Trying to automatically update all Nextcloud apps..."
-    # occ command can not be used due to the check_command() function.
-    sudo -u www-data php $NCPATH/occ app:update --all
-#    UPDATED_APPS="$(sudo -u www-data php $NCPATH/occ app:update --all)"
+    UPDATED_APPS="$(occ_command_no_check app:update --all)"
```

Comment:
Can't we use here the normal occ_command?

Reply:
> occ command can not be used due to the check_command() function.

No :)